dotcom-rendering/docs/contributing/code-style.md

```diff
@@ -221,3 +221,5 @@ const MyLink = ({ isActive }) => (
     <a className={cx({ [activeLink]: isActive }, link)}>Click Me</a>
 );
 ```
+
+### Don't use React.FC or equivalent
```

Comment:
Maybe more of a preference, but I tend to avoid negative contractions. [This is sometimes recommended](https://readabilityguidelines.co.uk/grammar-points/contractions/#1-avoid-negative-contractions).

```suggestion
### Do not use React.FC or equivalent
```